core/api/monitoring.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package v1beta1
```

Comment:
where the api versioning will be placed?

Reply:
`api` package under `kogito-cloud-operator` is a source of truth and it have version(`v1beta1`) subpackage under it.